api_tests/Test.cs

```diff
@@ -175,6 +177,89 @@ public void TestDeviceCapabilityQueries()
             Assert.AreEqual(set.Device as IReadable<Light> != null, set.Result);
         }
     }
-}
 
+	[Test()]
+	public void TestHouseOutput()
+	{
+		//Test valid URL and Device
+		//	to view correct POST:
+		//		go to http://postcatcher.in/ and click "Start testing your POST requests now"
+		//		copy the URL directly after "Content-Type: application/json" and replace the
+		//		URL below (http://postcatcher.in/catchers/5536e135f9562d0300003e57) with the
+		//		URL from postcatcher
+
+		const string url = "http://postcatcher.in/catchers/55439a9f51155a03000005a5";
```

Comment:
Nice - I'll have to catch this for future testing usage, this is really clever/useful.